BitwardenShared/UI/Auth/TwoFactorNotice/EmailAccess/EmailAccessView.swift

```diff
@@ -41,10 +41,13 @@ struct EmailAccessView: View {
 
     private var toggleCard: some View {
         VStack(alignment: .leading, spacing: 8) {
-            Text(LocalizedStringKey(Localizations.doYouHaveReliableAccessToYourEmail("person\u{2060}@example.com")))
-                .styleGuide(.body)
-                .foregroundColor(Asset.Colors.textPrimary.swiftUIColor)
-                .accessibilityHidden(true)
+            Text(LocalizedStringKey(Localizations.doYouHaveReliableAccessToYourEmailDescriptionLong(
+                // Adding the Word Joiner character (U+2060) in the middle of the email address
+                // keeps the markdown rendering from making the email address a tappable link.
+                store.state.emailAddress.replacingOccurrences(of: "@", with: "\u{2060}@")
```

Comment:
🤔 Really cool! Could we have a helper / extension method to do the replacement so it's easier to reuse and to see that something already exists to fix it?

Reply:
Aha, good call!